contracts/HATVaults.sol

```diff
@@ -91,7 +91,7 @@ contract  HATVaults is Governable, HATMaster {
     //pid -> amount
     mapping(uint256 => uint256) public governanceHatRewards;
     //pid -> PendingApproval
-    mapping(uint256 => PendingApproval) public pendingApprovals;
+    mapping(uint256 => SubmittedClaim) public submittedClaims;
```

Comment:
perhaps `claims` here instead of `submittedClaims`

Reply:
again, it's for claims that have been submitted by the committee